app/src/features/surveys/view/survey-spatial/components/animal/SurveySpatialAnimalCapturePopup.tsx

```diff
@@ -45,15 +49,17 @@ export const SurveySpatialAnimalCapturePopup = (props: ISurveySpatialAnimalCaptu
       closeButton={true}
       autoPan={true}
       eventHandlers={{
-        add: () => {
-          captureDataLoader.load(String(feature.id));
+        add: async () => {
+          const capture = await captureDataLoader.load();
+          // Fetch critter information to get the animal's nickname to display, which isn't included in the capture response
+          if (capture) animalDataLoader.load(capture.critter_id);
```

Comment:
We should include the brackets in if statements like this. Helps prevent mistakes, and also prevents a few weird edge cases that can crop up when writing them without brackets.

```
if (capture) {
  animalDataLoader.load(capture.critter_id);
}
```